charts/grafana/templates/grafana-deployment.yaml

```diff
@@ -35,6 +35,22 @@ spec:
       {{- if and (not .Values.backendSecretName) (not .Values.backendConnection) }}
       serviceAccountName: {{ template "grafana.bootstrapper.serviceaccount" . }}
       initContainers:
+        # TODO: add wait-for-db or similar to grafana container
```

Comment:
@sjmiller609 do we need to adjust this `if` logic here like we did on Houston? We'd always want this `initContainer` to run, not follow the same logic as the bootstrapper right?

Reply:
ah yes I think you are right